dashboard/visualization/plots.py

```diff
@@ -530,27 +530,67 @@ def concentration_plot(df: pd.DataFrame, reaction_type: str) -> go.Figure:
     return fig
 
 
-def plot_ic50(entry: dict) -> go.Figure:
+def plot_ic50(entry: dict, x: np.ndarray, y: np.ndarray) -> go.Figure:
```

Comment:
another issue: the plot and values stay the same regardless of the chosen compound (no errors)
![image](https://github.com/zuzg/drug-screening/assets/81712088/0f02f0b5-8451-409e-8d88-dce5007e74f4)

Reply:
> another issue: the plot and values stay the same regardless of the chosen compound (no errors) ![image](https://user-images.githubusercontent.com/81712088/275314868-0f02f0b5-8451-409e-8d88-dce5007e74f4.png)

hmm this looks like what was being generated before my PR as a placeholder, but on my branch its impossible to get this plot as the mock data used in here no longer exists in the codebase (for my branch)

\+ similarily for the Compounds data (you can see it in the `dashboard.pages.hit_validation.callbacks.on_selected_compound_changed` function that the data should be processed correctly (there are no placeholders)

Maybe you have a stale version of the branch?

Reply:
you are right! sorry, not sure how that happened